src/clusterfuzz/stacktraces/constants.py

```diff
@@ -646,10 +649,12 @@
     (UBSAN_MISALIGNED_ADDRESS_REGEX, 'Misaligned-address'),
     (UBSAN_NO_RETURN_VALUE_REGEX, 'No-return-value'),
     (UBSAN_NULL_ARGUMENT_REGEX, 'Invalid-null-argument'),
+    (UBSAN_IMPLICIT_CONVERSION_REGEX, 'Implicit-conversion'),
```

Comment:
@hogo6002 added this separately in another PR, so now these Implicit-conversion changes are now duplicate. Can you please remove?